hedera-mirror-rest/model/node.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2021-2024 Hedera Hashgraph, LLC
```

Comment:
Copyrights should match the years in which the file was active.

```suggestion
 * Copyright (C) 2024 Hedera Hashgraph, LLC
```

Reply:
Done.